addon/store.js

```diff
@@ -0,0 +1,48 @@
+import Ember from 'ember';
+import DS from 'ember-data';
+
+const keys = Ember.keys;
+const create = Ember.create;
+export const knownStores = [];
+
+export default DS.Store.extend({
+  init() {
+    this._super(...arguments);
+    knownStores.push(this._debugContainerKey);
```

Comment:
But you are closing over an array that is static? Doesn't the knownStores array leak between requests?